### PR TITLE
Remove remaining Gogs reference on locales and cmd

### PR DESCRIPTION
**Pull request by joubertredrat**
Remove Gogs mention on locale variable and cmd strings

### CODE REVIEW

File: cmd/admin.go [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Copyright 2016 The Gogs Authors. All rights reserved.
+// Copyright 2016 The Gitea Authors. All rights reserved.
```
Comment (lunny) [inline cmd/admin.go:2]: Please send this line top on the Gogs

File: cmd/dump.go [hunk 2]
```diff
@@ -1,4 +1,5 @@
 // Copyright 2014 The Gogs Authors. All rights reserved.
+// Copyright 2016 The Gitea Authors. All rights reserved.
```
Comment (lunny) [inline cmd/dump.go:2]: ^

File: cmd/serve.go [hunk 3]
```diff
@@ -1,4 +1,5 @@
 // Copyright 2014 The Gogs Authors. All rights reserved.
+// Copyright 2016 The Gitea Authors. All rights reserved.
```
Comment (lunny) [inline cmd/serve.go:2]: ^

File: main.go [hunk 4]
```diff
@@ -1,5 +1,5 @@
-// Copyright 2016 The Gitea Authors. All rights reserved.
 // Copyright 2014 The Gogs Authors. All rights reserved.
+// Copyright 2016 The Gitea Authors. All rights reserved.
```
Comment (lunny) [inline main.go:2]: ^

**Review by lunny — CHANGES_REQUESTED** [review]

File: cmd/admin.go (hunk 1 above)
Comment (joubertredrat) [reply cmd/admin.go:2]: @lunny the correct is ordered by year asc, not?

Comment (lunny) [reply cmd/admin.go:2]: I thinks it should be ordered by year desc?

Comment (joubertredrat) [reply cmd/admin.go:2]: @lunny on Golang projects I don't know. In PHP and Python projects we put on year asc.

Comment (joubertredrat) [reply cmd/admin.go:2]: @lunny Okay, I will revert position and after we can talk about this :+1:

**Comment by andreynering** [comment]
LGTM

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
LGTM

**Pull request merged**